pom.xml

```diff
@@ -69,6 +69,7 @@
         <module>extensions/server/kyuubi-server-plugin</module>
         <module>extensions/spark/kyuubi-spark-authz</module>
         <module>extensions/spark/kyuubi-spark-connector-common</module>
+        <module>extensions/spark/kyuubi-spark-connector-hive</module>
```

Comment:
please refer kudu connector to add it as a spark 3.3 specific module since we decided to support 3.3+ only

Reply:
ok~  and it looks like have a flaky test not related to this pr? please take a look, you can click [log](https://pipelines.actions.githubusercontent.com/serviceHosts/618b7dd5-55a0-4451-951d-ef3929531f08/_apis/pipelines/1/runs/27921/signedlogcontent/5?urlExpires=2022-08-22T06%3A37%3A02.1086683Z&urlSigningMethod=HMACV1&urlSignature=LKth2y9vIVX%2B4Yhhza8oaNjxpIkM6f4yVFZUMN6WAww%3D) and search keyword of the `support to interrupt the thrift request if remote engine is broken *** FAILED ***`